tests/test_helper/lxd.bash

```diff
@@ -0,0 +1,40 @@
+LXC_TEST_IMAGE=jammy
+DEPENDENCIES="jq"
+
+function start_containers() {
+    LXC_COUNT=$1
+    LXC_BASE_NAME="microovn-test"
+    ALL_CONTAINERS=""
+    for (( i = 0; i < LXC_COUNT; i++ )); do
+        CONTAINER_NAME="$LXC_BASE_NAME-$i"
+        echo "# Launching $CONTAINER_NAME" >&3
+        lxc launch -q ubuntu:$LXC_TEST_IMAGE $CONTAINER_NAME
+        lxc_exec $CONTAINER_NAME "apt update && apt install -y $DEPENDENCIES"
+        ALL_CONTAINERS+="$CONTAINER_NAME "
+    done
+
+    export ALL_CONTAINERS
```

Comment:
We should be very careful with global variables in library/utility code. I wonder if it would be as easy for the calling code to provide identifiers for instances to create or something similar?